packages/legend-application-studio/src/stores/EditorStore.ts

```diff
@@ -340,17 +340,23 @@ export class EditorStore implements CommandRegistrar {
     this.mode = val;
   }
 
-  *setGraphEditMode(graphEditor: GRAPH_EDITOR_MODE): GeneratorFn<void> {
+  *setGraphEditMode(
+    graphEditor: GRAPH_EDITOR_MODE,
+    entities?: Entity[],
```

Comment:
I understand why we need to do this, but this seems really hacky, especially when one considers the name of the function :(